lib/resources/example_json/r4_examples_immunization.rb

```diff
@@ -105,5 +105,62 @@ module Resources
         }
       ]
     }
+
+    R4_IMMUNIZATION_CREATE ||= {
+      "resourceType": "Immunization",
+      "status": "completed",
+      "occurrenceDateTime": "2019-06-30T10:35:00-05:00",
+      "vaccineCode": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/72",
```

Comment:
change all instances of 2c400054 to 0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca

Reply:
Updated the tenant key everywhere applicable.
https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-b4a3f148f7c74d28ca272418093bb464R116